### ISSUE TITLE
Consecutive if statements not working in a list

### ISSUE THREAD

**Issue by tugrulelmas** [egonl/SharpDocx#18]
I want  to create a list with conditions but when I use if statements consecutively, the result doesn't contain second item, even if the condition is true.

Here is a sample template:
```
1. 	<% if(true) { %>Foo<% } %>
2.	<% if(true) { %>Bar<% } %>
3.	Foo Bar
```
Result:
```
1. 	Foo
2.	Foo Bar
```

Expected Result:
```
1. 	Foo
2. 	Bar
3.	Foo Bar
```

**Comment by Kyou12138** [comment]
1. 	<% if(true) { %>Foo<% } %>
2.     <% Write(""); %>
3.	<% if(true) { %>Bar<% } %>
4.	Foo Bar
This may be normal.

**Comment by egonl** [comment]
Fix is included in NuGet package 2.0.3. Please let me know if this works for you.

**Comment by tugrulelmas** [comment]
The fix solved my problem. Thanks for your effort.
